### DISCUSSION TITLE
Does bun have a twitter profile that I can follow?

### DISCUSSION THREAD

**Discussion by johhansantana** [oven-sh/bun#296] [Q&A] [answered]
Title question 😅

**Comment by Murtatrxx** [comment]
Yes

**Reply by johhansantana** [reply]
I’m guessing this is because I didn’t specify it enough but could you share the twitter link?

**Reply by Murtatrxx** [reply]
https://twitter.com/ropoxdev

**Reply by johhansantana** [reply]
I’m going to say that I am super bad at asking questions. I wanted to ask about the Bun’s twitter profile 🥲

**Reply by Murtatrxx** [reply]
Looool

**Reply by Murtatrxx** [reply]
There's no atm

**Reply by johhansantana** [reply]
lol ok thanks 🙃

**Reply by logikaljay** [reply]
No bun specific twitter account, but follow Jarred.

https://twitter.com/jarredsumner

**Reply by WaldemarEnns** [reply]
@Murtatrxx for the future: Please don't unnecessarily confuse other readers/people that have the same question as johhansantana.
@logikaljay thank you for the reference!

**Reply by johhansantana** [reply]
> @Murtatrxx for the future: Please don't unnecessarily confuse other readers/people that have the same question as johhansantana.
> @logikaljay thank you for the reference!

I edited the title to better explain my question. Maybe it was because I didn’t explain it correctly.

**Reply by WaldemarEnns** [reply]
@johhansantana no, your question was totally fine ;)

**Reply by Murtatrxx** [reply]
lmao, there was "Do you guys have twitter account?" And I replied with my own.

**Reply by WaldemarEnns** [reply]
@Murtatrxx so in the context of _this_ repo - you thought he was asking about random users twitter accounts? :D

**Reply by Murtatrxx** [reply]
Why not?!

**Reply by WaldemarEnns** [reply]
Please don't take it too personally. It's just weird that someone would ask for **random** twitter accounts on a very specific repositories Q&A section.

**Comment by Bellisario** [comment, marked as answer]
Yes and no... currently the official Bun Twitter profile is @Jarred-Sumner personal profile.
You can see the source code of the site contain it's Twitter profile and not another:
https://github.com/Jarred-Sumner/bun/blob/118c1349183f66736572d03ff6847b0fd465b5ad/packages/bun-landing/page.tsx#L228

Anyway would be great to have it!

**Comment by Jarred-Sumner** [comment]
Bun should clearly get an official twitter which is not just my personal Twitter

**Reply by kosperera** [reply]
Unfortunately, `bun`, `bunsh`, `bunjs`, `bunofficial`, `officialbun` have been taken 👎

**Reply by Murtatrxx** [reply]
Hey I have @bunruntime name and I can transfer it

**Reply by Murtatrxx** [reply]
@Jarred-Sumner Are you interested? With @bunruntime (I'll maintain it anyway)
